packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -430,7 +430,14 @@ describe('App: Settings without cloud', () => {
 
       expect(browsers).to.have.length.greaterThan(1)
 
-      cy.contains(`browsers: ${browsers.filter((b) => b.name !== 'electron').map((b) => b.name).join(', ')}`)
+      cy.contains(`browsers: [`)
+      cy.contains(`name: 'chrome',`)
+      cy.contains(`family: 'chromium',`)
+      cy.contains(`channel: 'stable',`)
+      cy.contains(`displayName: 'Chrome',`)
+      cy.contains(`path: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome',`)
```

Comment:
Is this spec loading browsers from a fixture? I think this will fail on other OS's otherwise.

Reply:
I just removed it, so we can be safe!